doc/modules/cassandra/pages/managing/operating/hints.adoc

```diff
@@ -152,7 +152,11 @@ megabytes. |`128MiB`
 
 |`hints_compression` |Compression to apply to the hint files. If
 omitted, hints files will be written uncompressed. LZ4, Snappy, and
-Deflate compressors are supported. |`LZ4Compressor`
+Deflate compressors are supported. |`false`
```

Comment:
Sorry, I just noticed another thing, should this change? it seems wrong to change the default to false no?
```suggestion
Deflate compressors are supported. |`LZ4Compressor`
```

Reply:
Mmmm, I wonder when I did change that. Yeah, good catch, that's definitely an unwanted change.